src/javascripts/components/search.tracking.js

```diff
@@ -0,0 +1,82 @@
+function addToDataLayer (payload) {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push(payload)
+}
+
+function stripPossiblePII (string) {
+  // Try to detect emails and redact it.
+  string = string.replace(/\S*@\S*\s?/g, '[blocked]')
+  // If someone has typed in a number it's likely not related so redact it
+  string = string.replace(/0|1|2|3|4|5|6|7|8|9/g, '[blocked]')
```

Comment:
possibly `[0-9 ]+` might work better? Replaces strings of numbers and spaces in one go
https://regexper.com/#%5B0-9%20%5D%2B

Reply:
Good idea, I think I'm going to try and remove this code though (see https://github.com/alphagov/govuk-design-system/pull/1827#issuecomment-901013426).

Reply:
Okay, couldn't get that working, went with your suggestion :)

Reply:
Ah, `[0-9 ]+` won't work, as it will also match any amount of spaces... It should really be something like `([0-9]+[ ]?)+`